tensorrt_llm/llmapi/llm_args.py

```diff
@@ -425,6 +426,50 @@ class EagleDecodingConfig(DecodingBaseConfig):
     max_non_leaves_per_layer: Optional[int] = None
     eagle3_one_model: Optional[bool] = True
 
+    def __init__(self, **kwargs):
+        super().__init__()
+        for attr_name, attr_value in kwargs.items():
+            if attr_name == 'max_draft_len':
+                self.num_eagle_layers = attr_value
+            # Convert the data type of Eagle choice from str to List[List[int]]
+            if attr_name == 'eagle_choices' and not isinstance(
+                    attr_value, list):
+                print(
+                    "NOTE: The Draft token tree is still under development, PLEASE DO NOT USE IT !!!"
+                )
+                if isinstance(attr_value, str):
+                    attr_value = ast.literal_eval(attr_value.replace(" ", ""))
+                else:
+                    assert "Wrong eagle choices type"
```

Comment:
`assert` should take a bool. I think you meant to do this:

```
assert isinstance(attr_value, str), "Wrong eagle choices type"
```

Better yet, why not `raise ValueError()` here?